putaway_apply/__openerp__.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) Bubbles-iT (<http://www.bubbles-it.be>)
+#    Copyright (C) 2004 OpenERP SA (<http://www.openerp.com>)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>
+#
+##############################################################################
+{
+    "name" : "Putaway apply",
+    "version" : "0.1",
+    "author": "Bubbles-iT, Odoo Community Association (OCA).",
+    "category": 'Warehouse Management',
+    "description":' Usually Putaway strategy works on categories matched to the product on Shipment. This module does it on Product level.',
```

Comment:
can you move out the description to a `README.rst` file? 

Here's a template: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst